.teamcity/Ribasim_Linux/buildTypes/Ribasim_Linux_TestRibasimBinaries.xml

```diff
@@ -29,7 +29,7 @@ pixi --version]]></param>
 source /usr/share/Modules/init/bash
 
 module load pixi
-pixi run --environment=dev install
+pixi run install
```

Comment:
The `install` task no longer exists, right? And `install-ci`, installs Julia, which isn't needed here. So I guess this line can be removed. Can you also trigger TeamCity builds on this branch to make sure that still works?

Reply:
I removed it and triggered both Linux and Windows Ribasim. They worked.